tests/components/config/test_entity_registry.py

```diff
@@ -286,6 +290,41 @@ async def test_update_entity(hass, client):
         "reload_delay": 30,
     }
 
+    # UPDATE ENTITY OPTION
+    await client.send_json(
+        {
+            "id": 10,
+            "type": "config/entity_registry/update",
+            "entity_id": "test_domain.world",
+            "options_domain": "sensor",
+            "options": {"unit_of_measurement": "beard_second"},
+        }
+    )
+
+    msg = await client.receive_json()
+
+    assert msg["result"] == {
```

Comment:
Why is `beard_second` not an error ?

Reply:
Oh I guess that entity registry doesn't know about valid sensor options.

Reply:
yeah, that's right.